packages/contracts/deploy/L1/017-GovernanceToken.ts

```diff
@@ -0,0 +1,45 @@
+import '@kroma/hardhat-deploy-config'
+import { ethers } from 'ethers'
+import { DeployFunction } from 'hardhat-deploy/dist/types'
+
+import { predeploys } from '../../src'
+import {
+  assertContractVariable,
+  deploy,
+  getDeploymentAddress,
+} from '../../src/deploy-utils'
+
+const deployFn: DeployFunction = async (hre) => {
+  const zeroAddress = ethers.constants.AddressZero
+  const l1StandardBridgeProxyAddress = await getDeploymentAddress(
+    hre,
+    'L1StandardBridgeProxy'
+  )
+
+  await deploy(hre, 'L1GovernanceToken', {
+    contract: 'GovernanceToken',
+    args: [
+      l1StandardBridgeProxyAddress,
+      predeploys.GovernanceToken,
+      zeroAddress,
+    ],
+    isProxyImpl: true,
+    postDeployAction: async (contract) => {
+      await assertContractVariable(
+        contract,
+        'BRIDGE',
+        l1StandardBridgeProxyAddress
+      )
+      await assertContractVariable(
+        contract,
+        'REMOTE_TOKEN',
+        predeploys.GovernanceToken
+      )
+      await assertContractVariable(contract, 'MINT_MANAGER', zeroAddress)
+    },
+  })
+}
+
+deployFn.tags = ['L1GovernanceToken', 'setup', 'l1']
```

Comment:
Why don't we add a tag for this upgrade? e.g. burgundy.

Reply:
I added `tge` tag.